pkg/apis/devices.harvesterhci.io/v1beta1/usb_device_claim.go

```diff
@@ -0,0 +1,29 @@
+package v1beta1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:object:root=true
+
+type USBDeviceClaim struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec   USBDeviceClaimSpec   `json:"spec,omitempty"`
```

Comment:
`USBDeviceClaimSpec` serves no real purpose here. should we just drop it like the `usbdevice` crd?

Reply:
yeah, need to do that.